Deployment/Mirantis-Guidance.md

```diff
@@ -0,0 +1,274 @@
+
+    The node type must be Silver or Gold durability level, because:
+    - Bronze does not give you any guarantees about saving state information.
+    - Silver and Gold durability trap any changes to the scale set.
+    - Gold also gives you control over the Azure updates underneath scale set.
+
+    In case one node type is serving as public endpoint, to reduce downtime a DNS switch by redirecting the traffic to a new Azure Public IP (PIP) address with Azure Traffic Manager or to another region is needed to keep downtime at minimum. Multiple zone scenarios are covered safely by the configuration sfZonalUpgradeMode:Hierarchical. In this way, only one zone goes through the upgrade at a time and the traffic is routed through one Azure Load Balancer on Standard SKU to all zones.
+
+## Scenario 1: Customer is hosting Azure Service Fabric Node Type using deprecated Windows Server images with-Containers, but does NOT host containers in Docker as part of their overall applications
+
+### Scenario 1/Option 1: Full rebuild the cluster
+
+This scenario fits where availability loss is acceptable, and effort is less through automation.
+
+Steps
+1.	Full rebuild of Service Fabric cluster on a supported OS SKU without container support.
+a.	Example for the OS SKU name: Windows Server 2022 Datacenter
+2.	Re-deploy applications
+
+> :exclamation:
+> Please consider the option to recreate the cluster by only removing the Azure Virtual Machine Scale Sets (VMSS) and the Azure Service Fabric cluster resource. Creating just these two instances works well when you don’t automate the whole deployment.
+
+Documentation:
+- [Quickstart: Create a Service Fabric cluster using ARM template](https://docs.microsoft.com/en-us/azure/service-fabric/quickstart-cluster-template)
+- [How To: Rebuild Azure Service Fabric cluster (minimal version)](#)
+
+### Scenario 1/Option 2: Mitigate via OS SKU upgrade
+
+The mitigation by adding a new Service Fabric Node Type and migrating the workload has the best cost benefit and the lowest risk in production.
+
+#### Primary Node Type on deprecated OS SKU
+
+Upgrade primary SKU to supported OS SKU without container support by following the linked documentation.
+
+Documentation: 
+- [Scale up a Service Fabric cluster primary node type](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-scale-up-primary-node-type)
+
+#### Secondary Node Type on deprecated OS SKU
+
+Add new secondary Node Types with supported OS SKU without container support by following the linked documentation.
+        
+Documentation: 
+- [Scale a Service Fabric cluster out by adding a virtual machine scale set](https://docs.microsoft.com/en-us/azure/service-fabric/virtual-machine-scale-set-scale-node-type-scale-out)
+- [Scale up a Service Fabric cluster secondary node type](#)
```

Comment:
not sure if you meant to add this link since just above on line 105 we have link to add new node type.  Either way it's not a valid link.

Reply:
Link to scale up secondary node type will be added as soon as we have the page.